docs/app/pages/components/components-sections/tables/hover-actions/hover-actions.component.html

```diff
@@ -18,9 +18,15 @@
                 <ux-spark theme="chart2" [value]="document.complete" [inlineLabel]="document.complete" barHeight="3"></ux-spark>
             </td>
             <td>
-                <i class="hpe-icon hpe-view hover-action-icon" aria-label="View" uxTooltip="View" uxHoverAction></i>
-                <i class="hpe-icon hpe-share hover-action-icon" aria-label="Share" uxTooltip="Share" uxHoverAction></i>
-                <i class="hpe-icon hpe-trash hover-action-icon" aria-label="Delete" uxTooltip="Delete" uxHoverAction></i>
+                <button class="hover-action-btn" aria-label="View" uxTooltip="View" uxHoverAction>
```

Comment:
The default `type` for a button is submit, so on the offchance that this appears in a form it would be better to specify `type="button"`.